pkg/storaged/storage.scss

```diff
@@ -373,3 +373,13 @@ a.disabled {
     // 1em: margin between text and progress bar
     padding-inline-end: calc(var(--pf-v5-global--spacer--sm) + 3.5em + 1em) !important;
 }
+
+// FIXME: this makes the first card in anaconda mode inconsistent with other cards which have shadow
```

Comment:
Could you put a class on just the breadcrumb to tweak its shadow? Would that be more proper? I am afraid I don't understand what is improper about your solution here.

Reply:
Also, what about changing the breadcrumb visuals also in non-Anaconda mode to match what we are trying to achieve here?

Reply:
I found a way to control box shadow in all directions. I think I can fix it properly.

Reply:
I thought that box-shadow it not possible to control per-direction - but apparently it is. I will utilize this and we should be good.

Reply:
Ok it's hard. I suggest we get this as is in, unless @mvollmer has a better idea, and we can consider how we can improve it in the future?

Reply:
There is `hasShadowBottom`, could we set that to false?

Reply:
> There is `hasShadowBottom`, could we set that to false?

No, it is already false by default.